src/lib/Behat/BrowserContext/BookmarkContext.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\AdminUi\Behat\BrowserContext;
+
+use Behat\Behat\Context\Context;
+use Ibexa\AdminUi\Behat\Page\BookmarksPage;
+use Ibexa\AdminUi\Behat\Page\ContentViewPage;
+use PHPUnit\Framework\Assert;
+
+class BookmarkContext implements Context
+{
+    /** @var \Ibexa\AdminUi\Behat\Page\ContentViewPage */
+    private $contentViewPage;
+
+    /** @var \Ibexa\AdminUi\Behat\Page\BookmarksPage */
+    private $bookmarksPage;
+
+    public function __construct(ContentViewPage $contentViewPage, BookmarksPage $bookmarksPage)
+    {
+        $this->contentViewPage = $contentViewPage;
+        $this->bookmarksPage = $bookmarksPage;
+    }
+
+    /**
+     * @Given I bookmark the Content Item :path
+     */
+    public function bookmarkContentItem(string $path): void
+    {
+        $this->contentViewPage->setExpectedLocationPath($path);
+        $this->contentViewPage->verifyIsLoaded();
+        $this->contentViewPage->bookmark();
```

Comment:
Quick nitpick here: 'bookmark' is a verb & noun in english so maybe we could be more precise here - even something like `bookmarkItem` or `bookmarkContentItem`

Reply:
Indeed 🧐 Changed in https://github.com/ezsystems/ezplatform-admin-ui/pull/2038/commits/b9b1849622101766f34ca2ed77fd157f2b883a58 , thanks!